msal-netcore-angular.csproj

```diff
@@ -5,6 +5,7 @@
     <TypeScriptCompileBlocked>true</TypeScriptCompileBlocked>
     <TypeScriptToolsVersion>Latest</TypeScriptToolsVersion>
     <IsPackable>false</IsPackable>
+    <UserSecretsId>0152be93-5c53-4399-923a-6e9ab1fe8846</UserSecretsId>
```

Comment:
What are you using the user secret for? If we don't need it, I rather not have it yet, it might be confusing.

Reply:
I put my tenant config in it.  That way I don't have to remember not to commit those changes.

Reply:
OK, makes sense. Might actually be useful to add something in the Readme about that, how to setup tenant config in UserSecret so that they're not in the repository.

Reply:
great idea!  let me do that